opm/core/transport/implicit/JacobianSystem.hpp

```diff
@@ -101,7 +101,7 @@ namespace Opm {
                 typedef typename BaseVec::value_type VT;
                 ::std::transform(x.begin(), x.end(),
                                  y.begin(),
-                                 ::std::bind2nd(::std::multiplies<VT>(), a));
+                                 [&a](auto x) { return a * x; });
```

Comment:
This will render the `VT` type alias unused.  You should remove that too.

Reply:
> This will render the `VT` type alias unused. You should remove that too.

Update force-pushed.